examples/protobuf/Deadline/Server/SlowChatbot.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc.Features;
+using IceRpc.Protobuf;
+using VisitorCenter;
+
+namespace DeadlineServer;
+
+/// <summary>A SlowChatbot is an IceRPC service that implements Protobuf service 'Greeter'.</summary>
+/// <remarks>The slow chatbot always delays its responses by 1 second.</remarks>
+[ProtobufService]
+internal partial class SlowChatbot : IGreeterService
+{
+    public async ValueTask<GreetResponse> GreetAsync(
+        GreetRequest message,
+        IFeatureCollection features,
+        CancellationToken cancellationToken)
+    {
+        Console.WriteLine($"Dispatching greet request {{ name = '{message.Name}' }}");
```

Comment:
Since the rpc method name is 'Greet', I would print Dispatching Greet request ...

Same for other service classes.